lib/src/screens/contact/contact_list_page.dart

```diff
@@ -77,56 +77,101 @@ class ContactListPage extends BasePage {
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(left: 24),
+      child: Column(
+        children: [
+          Align(
+            alignment: Alignment.centerLeft,
+            child: TabBar(
+              controller: _tabController,
+              splashFactory: NoSplash.splashFactory,
+              indicatorSize: TabBarIndicatorSize.label,
+              isScrollable: true,
+              labelStyle: TextStyle(
+                fontSize: 18,
+                fontFamily: 'Lato',
+                fontWeight: FontWeight.w600,
+                color: Theme.of(context).appBarTheme.titleTextStyle!.color,
               ),
-              child: ContactListBody(contactListViewModel: contactListViewModel),
+              unselectedLabelStyle: TextStyle(
+                  fontSize: 18,
+                  fontFamily: 'Lato',
+                  fontWeight: FontWeight.w600,
+                  color: Theme.of(context).appBarTheme.titleTextStyle!.color?.withOpacity(0.5)),
+              labelColor: Theme.of(context).appBarTheme.titleTextStyle!.color,
+              indicatorColor: Theme.of(context).appBarTheme.titleTextStyle!.color,
+              indicatorPadding: EdgeInsets.zero,
+              labelPadding: EdgeInsets.only(right: 24),
+              tabAlignment: TabAlignment.center,
+              dividerColor: Colors.transparent,
+              padding: EdgeInsets.zero,
+              tabs: [
+                Tab(text: S.of(context).wallets),
+                Tab(text: S.of(context).contact_list_contacts),
+              ],
             ),
-          ],
-        ),
+          ),
+          Expanded(
+            child: TabBarView(
+              controller: _tabController,
+              children: [
+                _buildWalletContacts(context),
+                ContactListBody(
+                    contactListViewModel: widget.contactListViewModel,
+                    tabController: _tabController),
+              ],
+            ),
+          ),
+        ],
       ),
     );
   }
 
-  Widget buildTitle(
-      {required String title, Widget? trailingFilterButton, double topPadding = 20.0}) {
-    return Container(
-        padding: EdgeInsets.only(top: topPadding, bottom: 5.0),
-        child: Row(mainAxisAlignment: MainAxisAlignment.spaceBetween, children: [
-          Text(title, style: TextStyle(fontSize: 36)),
-          trailingFilterButton ?? Container()
-        ]));
+  Widget _buildWalletContacts(BuildContext context) {
+    final walletContacts = widget.contactListViewModel.walletContactsToShow;
+
+    return ListView.builder(
+      shrinkWrap: true,
+      itemCount: walletContacts.length * 2,
+      itemBuilder: (context, index) {
+        if (index.isOdd) {
+          return StandardListSeparator();
+        } else {
+          final walletInfo = walletContacts[index ~/ 2];
+          return generateRaw(context, walletInfo);
+        }
```

Comment:
for future instances, there is `ListView.separated` where it will do this automatically

Reply:
In this specific case, we need a separator at the bottom of the list. ListView.separated only provides a separator between items.